plugins/modules/azure_rm_backupazurevm.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Suyeb Ansari (@suyeb786), Pallavi Chaudhari(@PallaviC2510)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = \
+    '''
+---
+module: azure_rm_backupazurevm
+version_added: '1.1.0'
+short_description: Back up an Azure Virtual Machine using Azure Backup
+description:
+    - Back up an Azure VM using Azure Backup.
+    - Enabling/Updating protection for the Azure VM.
+    - Trigger an on-demand backup for a protected Azure VM.
+    - Stop protection but retain existing data.
+    - Stop protection and delete data.
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: true
+        type: str
+    recovery_vault_name:
+        description:
+            - The name of the Azure Recovery Service Vault.
+        required: true
+        type: str
+    resource_id:
+        description:
+            - Azure Virtual Machine Resource ID.
+        required: true
+        type: str
+    backup_policy_id:
+        description:
+            - Backup Policy ID present under Recovery Service Vault mentioned in recovery_vault_name field.
+        required: true
+        type: str
+    state:
+        description:
+            - Assert the state of the protection item.
+            - Use C(create) for enabling protection for the Azure VM.
+            - Use C(update) for changing the policy of protection.
+            - Use C(stop) for stop protection but retain existing data.
+            - Use C(delete) for stop protection and delete data.
+            - Use C(backup) for on-demand backup.
+        default: create
+        type: str
+        choices:
+            - create
+            - update
+            - delete
+            - stop
+            - backup
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
+author:
+    - Suyeb Ansari (@suyeb786)
+    - Pallavi Chaudhari (@PallaviC2510)
+
+'''
+
+EXAMPLES = \
+    '''
+    - name: Enabling/Updating protection for the Azure VM
+      azure_rm_backupazurevm:
+        resource_group: 'myResourceGroup'
+        recovery_vault_name: 'testVault'
+        resource_id: '/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/testVM'
+        backup_policy_id: '/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/microsoft.recoveryservices/vaults/testVault/backupPolicies/ProdPolicy'
```

Comment:
The sentence is too long, please shorten it.

Reply:
Adjusted string with column size 80 as per pep8 standards